lib/spy.js

```diff
@@ -91,59 +251,76 @@ module.exports = function (chai, _) {
   }
```

Comment:
Yes, I also think that method should be removed and this should be a part of a separate PR